src/library/commissioner_safe.hpp

```diff
@@ -217,6 +214,9 @@ class CommissionerSafe : public Commissioner
         struct event_base *mEventBase;
     };
 
+    // The EventBaseHolder needs to be the first member so that
```

Comment:
any reason why so ?

Reply:
Because any other modules depends on an the `event_base` object.